modules/EPMC.py

```diff
@@ -120,8 +128,8 @@ def main(cooccurrenceFile, outputFile, local=False):
         .withColumn('datatypeId', pf.lit('literature'))
 
         # Reorder columns:
-        .select(['datasourceId', 'datatypeId', 'targetFromSource', 'targetFromSourceId', 'resourceScore',
-                'diseaseFromSource', 'diseaseFromSourceMappedId', 'literature', 'textMiningSentences'])
+        .select(['datasourceId', 'datatypeId', 'targetFromSourceId', 'diseaseFromSourceMappedId',
+            'resourceScore', 'literature', 'textMiningSentences', 'pmcIds'])
```

Comment:
Align & rewrap to 120 characters

```suggestion
        .select(['datasourceId', 'datatypeId', 'targetFromSourceId', 'diseaseFromSourceMappedId', 'resourceScore',
                 'literature', 'textMiningSentences', 'pmcIds'])
```